src/utils/encryption.js

```diff
@@ -19,6 +19,9 @@ const getPrimordialHash = entropy => {
   return CryptoJS.SHA256(entropy).toString();
 };
 
+const getNextHash = (previousHash) =>
+    CryptoJS.SHA256(previousHash).toString();
```

Comment:
i have this in my pull: https://github.com/oysterprotocol/webinterface/pull/33/files#diff-00fda04a9612e699d02df392d92f2951R24